docs/testsuite_compatibility_matrix.md

```diff
@@ -0,0 +1,69 @@
+Rest.li Multi-language Compatibility Matrix
+--------------------------
+Following [Rest.li Test Suite Specification](testsuite_overview.md), we've added test suites for Python and Java.
+Based on these test suites, we've made the following compatibility matrices:
+
+### Supported Resources
+
+| Resource | Java | Python | Additional Information |
+|--------|------|----------------|----------------|
+| ActionSet | x | x | |
+| Association | x | | Python generates code using a template engine, which doesn’t support Association |
+| Collection | x | x | |
+| Simple | x | x | Simple resource is supported by Python, but Python requires a top-level "namespace". If no namespace is specified by the Java-generated restspec, it cannot generate request builders. |
+
+
+### Supported Resource Keys
+
+| Key Feature | Java | Python | Additional Information |
+|--------|------|----------------|----------------|
+| ComplexKey | x |  |In Java, ComplexResourceKey is a map of complex keys and params. In Python, complex key is supported, but only as a record, not a map with params |
+| Key With Union | x | x | In Python, Key with Union is supported, but union member cannot be complex key with params |
+| Query parmas | x | x |
+
+
+### Supported Data Templates in Test Suite Spec
+
+| Template | Java | Python | Additional Information |
+|--------|------|----------------|----------------|
```

Comment:
Not sure if it is possible to make the first column of these tables a bit wider to avoid word wrap

Reply:
I fix this by replacing spaces with nonbreaking spaces (&nbsp;) in the first column. This makes the first column wider, but also makes the source code less readable.